doc/source/developer_info.rst

```diff
@@ -7,15 +7,15 @@ Release Process
 Prepare Repo for Release
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. *Determine the version number for the release. **bliss-core** uses standard semantic versioning (Major.Minor.Patch).
+1. *Determine the version number for the release. **ait-core** uses standard semantic versioning (Major.Minor.Patch).
 
 * Major bumps are for large, non-backwards compatible changes
 * Minor bumps are for backwards compatible changes
 * Patch bumps are for incremental bug fixes, small releases, and end-of-sprint releases.
 
 2. *Update the project documentation to use the correct version names.* The following files needs to be updated with the correct version names:
 
-* `doc/source/conf.py <https://github.jpl.nasa.gov/bliss/bliss-core/blob/master/doc/source/conf.py>`_ - contains a **version** and **release** option. Both of these should be updated to point to the version number for this release.
+* `doc/source/conf.py <https://github.jpl.nasa.gov/ait/ait-core/blob/master/doc/source/conf.py>`_ - contains a **version** and **release** option. Both of these should be updated to point to the version number for this release.
```

Comment:
github.j.n.g should be replaced with public GH url, but in this case, this docs page can actually be removed and its contents redirected via #29 and #35.